sdk/spring/spring-integration-azure-eventhubs/src/main/java/com/azure/spring/integration/eventhubs/inbound/health/EventHusProcessorInstrumentation.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.integration.eventhubs.inbound.health;
+
+import com.azure.messaging.eventhubs.models.ErrorContext;
+import com.azure.spring.integration.instrumentation.Instrumentation;
+
+import java.time.Duration;
+
+/**
+ * EventHus health details entity class.
+ */
+public class EventHusProcessorInstrumentation implements Instrumentation {
```

Comment:
Can we combine the EventHusProcessorInstrumentation and ServiceBusProcessorInstrumentation as a ProcessorInstrumentation?

Reply:
Most can be merged, but the difference is in `Throwable getException();` method.